packages/checkout-storefront/tsconfig.json

```diff
@@ -13,6 +14,10 @@
     "declarationMap": false,
     "declarationDir": "types",
     "outDir": "dist",
-    "composite": false
+    "composite": false,
+    "allowJs": true,
+    "noEmit": true,
+    "incremental": true,
+    "jsx": "preserve"
```

Comment:
All of those settings are used by `tsconfig/next.js` - maybe we should change the extends to that config instead?

Reply:
Given it's a temporary solution, I'd rather keep it here so it could be easily removed.